src/components/units/ChatRoom/ChatRoom.styles.ts

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+
+	background-color: gray;
+`;
+
+export const ChatHeader = styled.div`
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: space-between;
+
+	padding: 0 23px;
+	height: 55px;
+	background-color: #8cabd9;
+	border-bottom: 0.5px solid #cdcdcd;
+`;
+
+export const HeaderBox = styled.div`
+	height: 30px;
+	display: flex;
+	align-items: center;
+	gap: 0 20px;
+`;
+
+export const SVGIcon = styled.img`
+	width: 18px;
+	height: 18px;
+`;
+
+export const Name = styled.div`
+	font-weight: 600;
+	font-size: 16px;
+	margin-left: 10px;
+	line-height: 60px;
+	cursor: pointer;
+`;
+
+export const ChatContainer = styled.div`
+	height: calc(100vh - 110px);
+	flex-grow: 1;
+	overflow-y: auto;
+	background-color: #8cabd9;
+	padding: 15px;
+`;
```

Comment:
디자인 면에서 스크롤 바 없애는 게 어떨까요?
```suggestion
&::-webkit-scrollbar {
    display: none;
  }
`;
```

Reply:
오 너무 좋은 방법인것 같아요. 디테일한 의견 감사합니다 !